js/repositories.bzl

```diff
@@ -8,7 +8,6 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 
 versions = struct(
-    aspect_bazel_lib = "1.1.0",
```

Comment:
Not using this anymore?

Reply:
there is a mix of both styles across our repos but this one is the outlier so just going with the standard style now